website/docs/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -8,29 +8,34 @@ description: >-
+| `data.attributes.name`                | string  |                                                                                          | The name of the variable set.                                                                                                                |
+| `data.attributes.description`         | string  | `""`                                                                                     | Text displayed in the UI to contextualize the variable set and its purpose.                                                                  |
+| `data.attributes.global`              | boolean | `false`                                                                                  | When true, HCP Terraform automatically applies the variable set to all current and future workspaces in the organization.                    |
+| `data.attributes.priority`            | boolean | `false`                                                                                  | When true, the variables in the set override any other variable values with a more specific scope, including values set on the command line. |
+| `data.relationships.workspaces`       | array   | `[]`   | Array of references to workspaces that the variable set should be assigned to. |                                                                                                                                              |
+| `data.relationships.projects`         | array   | `[]`   | Array of references to projects that the variable set should be assigned to.   |                                                                                                                                              |
+| `data.relationships.vars`             | array   | `[]`   | Array of complete variable definitions that comprise the variable set.         |                                                                                                                                              |
+| `data.relationships.parent`           | object  | Organization that the  variable set belongs to                                           | The parent that owns this variable set. If the parent is a project, `data.attributes.global` must be `false`.                                      |
+| `data.relationships.parent.data.type` | string  | `"organizations"`                                                                        | The resource type of the parent that owns this variable set. Valid values are `organizations` or `projects`.                            |
+| `data.relationships.parent.data.id`   | string  | ID of organization that the variable set belongs to.                                     | The ID of the parent that owns the variable set.                                                                                       |
```

Comment:
I believe its the org name eg. tfc-enterprise-providers-test indead of an id like org-def4fd44re. Would calling it ID be confusing?

Reply:
That makes sense, I was confused by it initially too. 😅  Will make this change!